src/main/kotlin/no/nav/familie/ef/sak/api/avstemming/AvstemmingController.kt

```diff
@@ -0,0 +1,20 @@
+package no.nav.familie.ef.sak.api.avstemming
+
+import no.nav.familie.ef.sak.repository.domain.Stønadstype
+import no.nav.familie.ef.sak.task.GrensesnittavstemmingTask
+import no.nav.familie.kontrakter.felles.Ressurs
+import no.nav.familie.prosessering.domene.Task
+import no.nav.security.token.support.core.api.ProtectedWithClaims
+import org.springframework.web.bind.annotation.*
+
+@RestController
+@RequestMapping("/api/grensesnittavstemming")
+@ProtectedWithClaims(issuer = "azuread")
+class AvstemmingController (private val grensesnittavstemmingTask: GrensesnittavstemmingTask) {
+
+    @PostMapping("{stønadstype}")
+    fun utløsGrensesnittavstemming(@PathVariable stønadstype: Stønadstype, @RequestBody grensesnittavstemmingDto: GrensesnittavstemmingDto): Ressurs<Task> {
```

Comment:
Hvordan brukes responen her i frontend? Trenger man å returnere Task?

Reply:
Avstemmingen ska skje automatiskt så har inget med frontend å gjøre. Vi trenger dette endepunktet for å testa i preprod, men kommer også trenge det for att trigge førsta tasken når vi vel går i prod.

Reply:
Vi tenkte att det er grejt att returnere tasken slik att vi slipper koppla oss mot databasen.